introduction.tex

```diff
@@ -45,14 +45,31 @@ \section{Background}
 
 The three grand challenges can be summarized as a need to
 identify and plan for a ``most promising'' fuel cycle, while
-accurately calculating its impacts. 
-A simulation tool capable of accurately calculating the metrics
-of a fuel cycle transition scenario is essential for solving these challenges.
+accurately calculating its impacts.
+In 2011, the \gls{DOE} commissioned a report
+in order to plan the U.S. nuclear future.
+The report by Wigeland et al. identified potential fuel cycle groups and compared them
+to find the most `promising' fuel cycle \cite{wigeland_nuclear_2014}.
+The objective of the evaluation was to provide information about the
+potential benefits and challenges of nuclear fuel cycle options, in order to
+guide \gls{DOE} \gls{FCRD} program.
 
-This work will demonstrate capabilities of a system-level
-analysis tool, \Cyclus. The modeling capabilities of \Cyclus 
-will aid in the effort of solving
-the grand challenges mentioned above.
+However, this study was done for a static fuel cycle, where the
+material flow and reactor deployment are at equilibrium.
+Therefore the study did not take into account the dynamic
+changes in fuel demand and reactor deployment as well as the
+previously existing fleet. Thus, for a more comprehensive
+and realistic
+evaluation of a fuel cycle, fuel cycle analyses
+must include the transition from a previous fleet to
+the intended fuel cycle.
+
+Given this gap, a simulation tool capable of accurately
+calculating the metrics of a fuel cycle transition scenario
+is essential for solving these grand challenges. This work
+will demonstrate capabilities of a system-level analysis
+tool, \Cyclus. The added capabilities in this work will
+allow \Cyclus to model \gls{NFC} transition scenarios.
```

Comment:
Cyclus was able to model transitions before this work (as is clear, since accomplishing these transitions has required no actual changes to the framework itself). Please replace this with a more accurate depiction of your contribution with this work (benchmarking, tools for ease-of-use, tools for improved reactor modeling)...